core/backend/src/TileStorage.ts

```diff
@@ -44,7 +44,12 @@ export class TileStorage {
     if (this._initializedIModels.has(iModelId))
       return;
     if (!(await this.storage.baseDirectoryExists({ baseDirectory: iModelId }))) {
-      await this.storage.createBaseDirectory({ baseDirectory: iModelId });
+      try {
+        await this.storage.createBaseDirectory({ baseDirectory: iModelId });
+      } catch (e: any) {
+        if(e.statusCode !== 409)
```

Comment:
Is it worth documenting what a 409 is and why it might occur?

Reply:
Yes.

Reply:
It's a standard HTTP status code, but I added a comment explaining why this happens